src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -239,5 +239,17 @@ private async Task<HttpResponseMessage> SendRequest(string api, StringContent re
                 _syncRoot.Release();
             }
         }
+
+        private static GitHubUser ResolveGitHubUserConflict(GitHubUser a, GitHubUser b)
+        {
+            // Pick the user with a valid Id
+            if (a.Id is null)
+                return b;
+            if (b.Id is null)
+                return a;
+
+            // otherwise pick a random one
+            return a;
```

Comment:
what about using latest by comparing update_at?

Reply:
It is not expected to enter here, thus I'm using the simplest approach.

Reply:
what about the user rename case?

---
In reply to: [383781441](https://github.com/dotnet/docfx/pull/5560#discussion_r383781441) [](ancestors = 383781441)